src/CSharpMongoMigrations.Tests/Migrations/MigrationTests.cs

```diff
@@ -36,5 +47,20 @@ public void UseDatabaseFact()
             // Assert
             migration.Db.Should().Be(db.Object);
         }
+
+        [Fact]
+        public void ConditionalMigrationFact()
```

Comment:
It's not required being tested. You set up the stub methods and later test them. It's unnecessary. Could you please remove this test?

Reply:
Removed.